core/include/detray/core/detail/single_store.hpp

```diff
@@ -63,7 +63,7 @@ class single_store {
     /// Construct with a specific memory resource @param resource
     /// (host-side only)
     template <typename allocator_t = vecmem::memory_resource>
-    requires(!detail::is_device_view_v<allocator_t>) DETRAY_HOST
+    requires(!concepts::device_view<allocator_t>) DETRAY_HOST
```

Comment:
Maybe this would be clearer as:

```suggestion
    requires(std::derived_from<allocator_t, std::pmr::memory_resource>)
```

Reply:
Done